Content.Server/Pinpointer/PinpointerSystem.cs

```diff
@@ -22,6 +25,9 @@ public override void Initialize()
 
         SubscribeLocalEvent<PinpointerComponent, ActivateInWorldEvent>(OnActivate);
         SubscribeLocalEvent<FTLCompletedEvent>(OnLocateTarget);
+        SubscribeLocalEvent<BeforeEnterPausedMapEvent>(OnBeforeEnterPausedMap);
+        SubscribeLocalEvent<AfterExitPausedMapEvent>(OnAfterExitPausedMap);
+        SubscribeLocalEvent<SpecialRespawnEvent>(OnSpecialRespawn);
```

Comment:
this running on every broadcast seems bad, feels like pinpointer targets should get some marker component so these can be directed subscriptions